### DISCUSSION TITLE
Conditionally Show Routes

### DISCUSSION THREAD

**Discussion by skylerkatz** [spatie/laravel-ignition#78] [Ideas]
Hello!

I was wondering if you would be interested in a PR that would only register the `_ignition` routes if the app was in `APP_DEBUG=true`, or even a dedicated `IGNITION_ROUTES_ENABLED` config value.

I bring this up because we have been looking through our production logs and noticed many different visits to the routes that ignition registers. I know that you make a check for `ensureLocalEnvironment` in the `ExecuteSolutionController` but no such checks are made in the other controllers.

If this is something you are interested in, I would be happy to submit a PR.

My current _workaround_ is to just update the route prefix to something else, but I am still not sure we need the routes at all in production.

Thanks!

**Comment by nandi95** [comment]
`IGNITION_ROUTES_ENABLED` would give better control so I think that should be preferred.
